### ISSUE TITLE
package.json "module" property points to file that is not published on npm

### ISSUE THREAD

**Issue by gobengo** [digitalbazaar/zcap-context#8]
https://github.com/digitalbazaar/zcap-context/blob/main/package.json#L6C3-L6C37

When bundling with vite I saw this error
```
[vite] Internal server error: Failed to resolve entry for package "@digitalbazaar/zcap-context". The package may have incorrect main/module/exports specified in its package.json.
```

Sure enough, you can see on npm that whereas the package.json links to a `./dist/context.esm.js` file, on npm that directory only contains a `context.js` (sans `esm`)
<img width="347" alt="Screenshot 2024-11-13 at 9 48 16 AM" src="https://github.com/user-attachments/assets/3744f004-2455-41fc-9d4e-6ae6345f2e69">

**Comment by gobengo** [comment]
I proposed a fix in https://github.com/digitalbazaar/zcap-context/pull/9

**Comment by davidlehn** [comment]
Fixed in v2.0.1.
